### DISCUSSION TITLE
How to add Document Category

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [madrecha/jamku#627] [QA] [answered]
1. How to add Inward category
2. How to add outward category
3. How to add Document Request category

**Comment by Divwork92** [comment, marked as answer]
You can create document category from office configuration - document category - new 

1. For inward category, go to  office configuration - document category 
    click on new
    select the category type as inward, 
    select color, add category name eg. inward remittance, inward invoice, etc
    click on save  

2. For outward category go to  office configuration - document category
    click on new
    select the category type as outward, 
    select color, add category name eg.  outward invoice, etc
    click on save 

3. For document request category go to  office configuration - document category
    click on new
    select the category type as request, 
    select color, add category name 
    click on save 

**Once you have created the document category, you will be able ti view in a drop down under document module
